src/mame/layout/checkma5.lay

```diff
@@ -326,11 +326,9 @@
 		<bezel name="lamp5" element="BET2" inputtag="P1" inputmask="0x20">
 			<bounds x="1.09" y="3.28" width="0.31" height="0.24" />
 		</bezel>
-		<!-- FIXME - missing element definition
-		<bezel name="lamp4" element="BET3" inputtag="P1" inputmask="0x10">
+		<bezel name="lamp4" element="BET3C" inputtag="P1" inputmask="0x10">
```

Comment:
I don't think this is really correct - for other things in this layout there are separate `BETn` and `BETnC` elements.  I think the `BET3` definition is actually missing.